docs/compose/ChipGroup/README.md

```diff
@@ -56,3 +74,59 @@ BpkSingleSelectChipGroup(
     type = BpkSingleChipGroupType.Wrap,
 )
 ```
+
+Example of a Rail MultiSelectChipGroup Without Sticky Chip:
+
+```Kotlin
+import net.skyscanner.backpack.compose.chipgroup.multiple.BpkMultiChipGroupType
+import net.skyscanner.backpack.compose.chipgroup.multiple.BpkMultiChipItem
+import net.skyscanner.backpack.compose.chipgroup.multiple.BpkMultiSelectChipGroup
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+
+val state = viewModel.uiState.collectAsStateWithLifecycle()
+
+BpkMultiSelectChipGroup(
+    chips = state.value,
```

Comment:
rather than `state.value` can you define a list of chip items with a static selected variable? 
you can add a note at the top that you'd generally drive these from a viewmodel. we generally keep our components agnostic from architecture etc